javascript/javascript1/week4/script.js

```diff
@@ -91,7 +85,7 @@ console.log(getReply("What day is it today")); // output Today is 16 June 2022
 console.log(getReply("what is 3 + 3")); //output The result is 6 
 console.log(getReply("what is 8 * 3")); // output The result is 24
 console.log(getReply("what is 50 / 4")); // output The result is 12.5
-console.log(getReply("Set a timer for 4 minutes")); //output undifined not sure why
+console.log(getReply("Set a timer for 4 minutes")); //output Not the right command
```

Comment:
Copy-paste the whole script.js file to https://beautifytools.com/javascript-validator.php
and check your syntax. 
<img width="1783" alt="Screenshot 2022-06-20 at 13 59 33" src="https://user-images.githubusercontent.com/10253713/174596988-1bde583a-9948-46cf-ba34-f2b0d2ffa538.png">

You also didn't close `}` after the switch.